gabarit/template_nlp/nlp_project/package_name/models_training/model_class.py

```diff
@@ -350,22 +350,18 @@ def get_and_save_metrics(self, y_true, y_pred, x=None, series_to_add: Union[List
         self.logger.info('--------------------------------')
 
         # Metrics file
-        df_stats = pd.DataFrame(columns=['Label', 'F1-Score', 'Accuracy',
-                                         'Precision', 'Recall', 'Trues', 'Falses',
-                                         'True positive', 'True negative',
-                                         'False positive', 'False negative',
-                                         'Condition positive', 'Condition negative',
-                                         'Predicted positive', 'Predicted negative'])
+        dict_df_stats = {}
```

Comment:
Is there a specific reason to use a dict later transformed into a df rather than replace 'append' with 'concat' as suggested by both the pandas issue and the original gabarit issue?

Reply:
Because the output of the function _update_info_from_c_mat which populates df_stats is a dictionary